injestion/external_ingest.py

```
@@ -0,0 +1,109 @@
+load_dotenv()
+
+KAFKA_BROKER = os.getenv("KAFKA_BROKER")
+WS_TOPIC = os.getenv("WS_TOPIC")
+DB_TOPIC = os.getenv("DB_TOPIC")
+EXTERNAL_TOPIC = os.getenv("EXTERNAL_TOPIC")
+REDIS_HOST = os.getenv("REDIS_HOST")
+
+try:
+    REDIS_PORT = int(os.getenv("REDIS_PORT"))
+except ValueError:
+    raise ValueError("REDIS_PORT must be a valid integer")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)